docs/data-management.rst

```diff
@@ -81,15 +91,19 @@ Browsing and rendering
 
 #.  These represent imported Images. The original Images are stored on the server and the generated thumbnails allow us to browse them.
 
-#.  Bio-Formats,\ https://www.openmicroscopy.org/bio-formats/\ , is used to read the pixel-data and metadata from over 150 different image formats, including multi-z timelapse images with many channels, they are referenced as 5D Images. Large pathology and medical images are also supported.
+#.  `Bio-Formats <https://www.openmicroscopy.org/bio-formats/>`_ is used to read the pixel-data and metadata from over 150 different image formats, including multi-z timelapse images with many channels, they are referenced as 5D Images. Large pathology and medical images are also supported.
+
+#.  For HCS data, the layout of the OMERO.web is a bit different. The HCS data are usually organized in following manner: ``Images`` are contained in ``Wells``, ``Wells`` are contained in ``Plates`` and ``Plates`` are organized in ``Screens``. The screenshot below shows the typical layout of a ``Plate`` in OMERO.web, where the ``Wells`` are organized in rows and columns. One ``Well`` is selected in central pane and it contains 4 ``Images`` whose thumbnails are displayed below the central pane. The bottom-left corner shows positions of the images inside that ``Well``.
```

Comment:
I don't know if it's worth mentioning runs.

Reply:
Fixed in https://github.com/ome/omero-guide-introduction/pull/4/commits/278615381ae0a95a61dedf4c441951e025f4777a